Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,47 @@
 ﻿using System;
+using System.Diagnostics;
 
 namespace BootCamp.Chapter
 {
-    class Program
+    public static class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name;
+            string surname = "";
+            int age = 0;
+            double weight = 0;
```

Comment:
Declaring variables here (no matter if initialized or not) will force you or another programmer to memorize the types later on. Also you do not know when this variable will be used first, except you search for it. Therefor I heavily recommend introducing variables as late as neccessary to avoid confusions later on.

Reply:
Hi, thank you for the review. Understood. I updated the code. Hope it will be okay now.